ui/nuxeo-pagination-controls.js

```diff
@@ -68,7 +107,11 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             disabled$="[[_isFirst(page)]]"
           >
           </paper-icon-button>
-          <span class="controls">[[page]]/[[numberOfPages]]</span>
+          <div class="controls">
+            <nuxeo-select options="[[_computePageOptions(numberOfPages)]]" selected="{{page}}" dynamic-align>
+            </nuxeo-select>
+            <span class="total">/ [[numberOfPages]]</span>
+          </div>
           <paper-icon-button
```

Comment:
Currently, in aspera ftests, we have to use the `icon` in order to select the correct `paper-icon-button` to navigate pages. It would be useful to take take this opportunity and add names or ids to each `paper-icon-button`, if you don't mind 🙂

Reply:
yes! done!

Reply:
Perfect, thanks! :)